test/es-module/test-esm-loader-hooks.mjs

```diff
@@ -195,4 +195,43 @@ describe('Loader hooks', { concurrency: true }, () => {
     assert.strictEqual(code, 0);
     assert.strictEqual(signal, null);
   });
+
```

Comment:
Let’s add a test for the simplest case, `./node --loader ./test/fixtures/empty.js`.

Reply:
Tests that deal with REPL are simply not convenient and hard to not have flaky. Let's not do that?